CHANGELOG.md

```diff
@@ -8,161 +8,164 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [PR-139](https://github.com/itk-dev/economics/pull/139)
+  1890: Added check that issue exists before adding worklog to database.
+
```

Comment:
Why are you suddenly changing `*` to `-`?

Reply:
Markdown linter was complaining

Reply:
I think it complained only because you used `-` in line 11. Please use `*` to match the rest of the file.

Reply:
I think you are right